src/io/uring/Ring.hxx

```diff
@@ -18,6 +18,16 @@ namespace Uring {
 class Ring {
 	struct io_uring ring;
 
+// io_uring_setup() flags supported by Linux kernel 5.6
+unsigned iouring_setup_mask = (
```

Comment:
- bad indentation
- please use doxygen-style comments for API documentation
- the parentheses are useless
- the variable should be `static constexpr`; a regular class field adds unnecessary overhead
- the variable should have a better name; this is *not* the general io_uring setup mask, after all, but the name suggests tha